deepchem/trans/transformers.py

```diff
@@ -994,7 +994,7 @@ def transform_array(self, X, y, w):
             [self.outputs], feed_dict={self.inputs: X_batch})[0]
         X_out.append(output)
         num_transformed = num_transformed + X_batch.shape[0]
-        print('%i samples transformed' % num_transformed)
+        # print('%i samples transformed' % num_transformed)
```

Comment:
Would you mind uncommenting this line? (If the print-out is bad for a large dataset like ANI-1, perhaps add a flag to turn off the printout)